src/Data/Semigroup/Instances.hs

```diff
@@ -5,8 +5,10 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 module Data.Semigroup.Instances where
 
+#if !(MIN_VERSION_fingertree(0,1,2) || MIN_VERSION_base(4,9,0))
```

Comment:
fingertree has only an instance for newer base, did anyone tried to persuade Ross to have conditional dependency on `semigroups`, so we can just have `fingertree >= 0.1.3` or so lower bound?

Reply:
Alas, `fingertree` is one of those libraries that has extremely minimal dependencies (only `base`). I imagine folks were too timid to ask him to tack on extra dependencies.

Reply:
Not even
```
if !impl(ghc >= 8.0)
  build-depends: semigroups
```
?

Reply:
If you want to make a pull request, then go ahead :)